migrations/versions/e46b30dd3ff5_automatic_collections.py

```diff
@@ -18,27 +17,39 @@
 
 
 def upgrade():
-    # TODO: Maybe add some CHECK constraints around some of these fields? Or just
-    # enforce in code upon creation.
     op.create_table(
-        "automatic_collection_definition",
+        "automatic_collection",
         sa.Column(
             'collection_id',
             sa.Integer,
             sa.ForeignKey('collections.id', ondelete='CASCADE'),
             primary_key=True,
         ),
         sa.Column(
-            'query',
-            sa.ARRAY(sa.String),
+            'keyword_query',
+            sa.String,
+        ),
+        sa.Column(
+            'author_query',
+            sa.String,
+        ),
+        sa.Column(
+            'title_query',
+            sa.String,
+        ),
+        sa.Column(
+            'subject_query',
+            sa.String,
```

Comment:
This is a good idea to make the type of queries the user puts in very clear for us backend engineers.